x-pack/plugins/actions/server/create_execute_function.ts

```diff
@@ -29,45 +22,27 @@ export interface ExecuteOptions {
   apiKey: string | null;
 }
 
+export type ExecutionEnqueuer = (
+  savedObjectsClient: SavedObjectsClientContract,
+  options: ExecuteOptions
+) => Promise<void>;
+
 export function createExecuteFunction({
```

Comment:
I wonder if this should now be called `createExecutionEnqueuerFunction` (+ file name, etc)?